whimsical-woodpeckers/anon/views.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+# from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from anon.models import AnonUser
+import random
+import string
+from django.utils import timezone
+# Create your views here.
+
+
+def random_string(length):
+    i = 0
+    pw = ""
+    while i < length:
+        pw = pw + random.choice(string.printable)
+        i += 1
+    return pw
```

Comment:
random.choice does not scale well. it would be better to use random.choices here.